client/components/AssignDoi/assignDoi.scss

```diff
@@ -2,4 +2,12 @@
 	> p {
 		margin: 0 0 12px 0;
 	}
+
+	.dropdowns {
+		display: flex;
+
+		> .bp3-form-group.input-field-component {
```

Comment:
Do we need both classes for specificity here? If not, I would it is "conventional" in scare quotes for us to just reference the PubPub class name (`input-field-component`). If so, a comment saying that it's a specificity hack is really helpful to have later.

Reply:
Yep this was for specificity. I'll throw a comment in!